index.html

```diff
@@ -0,0 +1,169 @@
+          <p class="card-title">Male</p>
+        </div>
+        <div class="card-description-container">
+          <p class="card-description">Your favorite outfit in our catalog with variety in Prada, Gucci, Balenciaga</p>
+        </div>
+      </div>
+      <!-- Third card -->
+      <div class="main-card-container">
+        <div class="card-icon-container">
+          <i class="fas fa-star"></i>
+        </div>
+        <div class="card-title-container">
+          <p class="card-title">Ranking</p>
+        </div>
+        <div class="card-description-container">
+          <p class="card-description">Find out about the most controversial and outgoing outfits on our list</p>
+        </div>
+      </div>
+      <!-- Fourth card -->
+      <div class="main-card-container">
+        <div class="card-icon-container">
+          <i class="fas fa-dumbbell"></i>
+        </div>
+        <div class="card-title-container">
+          <p class="card-title">Sporty</p>
+        </div>
+        <div class="card-description-container">
+          <p class="card-description">Unisex sportswear for all occasions Adidas, Asics, Nike, Reebok</p>
+        </div>
+      </div>
+      <!-- Fith card -->
+      <div class="main-card-container">
+        <div class="card-icon-container">
+          <i class="fas fa-info-circle"></i>
+        </div>
+        <div class="card-title-container">
+          <p class="card-title">About</p>
+        </div>
+        <div class="card-description-container">
+          <p class="card-description">Learn more about our work and paid services, contacts, shipments</p>
+        </div>
+      </div>
+    </div>
+    <div class="main-button-container">
+      <a href="#" aria-label="link" class="main-button"> Join Global Outfit <span class="numbers">2022</span></a>
+    </div>
+  </div>
+</section>
+<!-- Project speaker section -->
+<section id="speaker-section">
+  <div id="speaker" class="speaker-container">
+    <div class="main-title-container">
+      <h4 class="main-title-b">Outfit Speakers</h4>
+      <div class="line"></div>
+      </div>
+      <div class="speaker-outfit-card-container"></div>
+      <div id="partners" class="partner-container-b">
+        <h4 class="main-title">Partner</h4>
+          <div class="line"></div>
+          <div class="partners-b">
+            <img class="partners-logo" src="src/tiktok-bg.png" alt="partner">
+            <img class="partners-logo" src="src/nivea-bg.png" alt="partner">
+            <img class="partners-logo" src="src/nike-bg.png" alt="partner">
+            <img class="partners-logo" src="src/amazon-bg.png" alt="partner">
+            <img class="partners-logo" src="src/polaris-bg.png" alt="partner">
+          </div>
+          <div class="footer-partner-b">
+            <div class="footer-a"><img src="src/project-logo.webp" alt="footer partner"></div>
+            <div class="footer-b">Creative Global Outfit<br>権利予約盗作は、著作権を帰属させることによってコミットされます</div>
+          </div>
```

Comment:
- [ ] The footer is well placed. I think it will be good for semantic purposes and easy reading to use the `footer` tag in replace of the `div` tag.